controllers/awsmachine_controller.go

```diff
@@ -18,17 +18,18 @@ package controllers
 
 import (
 	"context"
-
```

Comment:
nit: this space should remain to separate stdlib imports from other imports, `reflect` should also be in the above grouping with `context`